lib/string-checker.js

```diff
@@ -84,18 +85,19 @@ StringChecker.prototype = {
     checkString: function(str, filename) {
         filename = filename || 'input';
         str = str.replace(/^#!?[^\n]+\n/gm, '');
+        var esprimaOptions = extend({
```

Comment:
Also, `_extend` isn't documented Node API. Might be better just to use a `forin` here.